.travis.yml

```diff
@@ -6,4 +6,4 @@ install:
   - pip install -r requirements.txt
 # command to run tests
 script:
-- PYTHONPATH=. pytest tests/test_response.py
\ No newline at end of file
+- PYTHONPATH=. pytest tests/
```

Comment:
This was an oversight from when I originally build up the framework.  I  ran tests locally and it worked, but there were some recent changes to classes, which caused things to fail that I needed to fix.